content/python/concepts/sets/terms/update/update.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: '.update()'
+Description: 'Updates the current set, by adding items from another iterable(set, list, tuple or dictionary), removing any duplicates.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Collections'
+  - 'Functions'
+  - 'Methods'
+  - 'Sets'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
```

Comment:
```suggestion
---
Title: '.update()'
Description: 'Updates the original set by adding elements from another set.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Collections'
  - 'Methods'
  - 'Sets'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---
```

Reply:
Modifications:
1. The description should be <160 characters and short.
2. Let's use one of the tags, either functions or methods. Since your entry addresses the `.update()` as a `method`, removed the `Functions` tag.